CONTRIBUTORS.md

```diff
@@ -322,7 +323,7 @@ Place: São Paulo, SP, Brazil </br>
 Coding Experience: Python</br>
 Email: beatriz.uezu@gmail.com </br>
 
-﻿Name: [Hugo Nunes](https://github.com/hugonun)<br/>
+Name: [Hugo Nunes](https://github.com/hugonun)<br/>
```

Comment:
To avoid merge conflicts, would you mind reversing these lines?

Reply:
I have edited the file already, hopefully it works

Reply:
@wzhouwzhou i have merged your develop branch to my branch, maybe it would solve the merge conflicts, if not, I can just revert it to the previous commit

Reply:
Seems like it's still trying to update files like _assemble/check.coffee

Reply:
Do you think you can just make a new branch, copy only ejbarba_hex_to_string, and only your bio (without touching other people's work), and just make a clean PR?